### PR TITLE
starting oracle linux

### PR DESCRIPTION
**Pull request by chayim**

### CODE REVIEW

File: bin/getepel [hunk 1]
```diff
@@ -4,6 +4,7 @@ PROGNAME="${BASH_SOURCE[0]}"
 HERE="$(cd "$(dirname "$PROGNAME")" &>/dev/null && pwd)"
 READIES=$(cd $HERE/.. && pwd)
 . $READIES/shibumi/defs
+DIST="$(./deps/readies/bin/platform --dist)"
```
Comment (rafie) [inline bin/getepel:7]: Better to:
```
DIST="$(source /etc/os-release; echo $ID;)"
```
As `platform` requires Python and maybe it's better not to require it here.

File: paella/platform.py [hunk 2]
```diff
@@ -328,7 +328,7 @@ def is_debian_compat(self):
         return self.dist in ['debian', 'ubuntu', 'linuxmint', 'rasbpian']
 
     def is_redhat_compat(self):
-        return self.dist == 'redhat' or self.dist == 'centos' or self.dist == 'amzn'
+        return self.dist == 'redhat' or self.dist == 'centos' or self.dist == 'amzn' or self.dist == 'ol'
```
Comment (rafie) [inline paella/platform.py:331]: Maybe `self.dist in ['redhat', 'centos', 'amzn', 'ol']`?

**Review by rafie — DISMISSED** [review]

File: bin/getepel (hunk 1 above)
Comment (chayim) [reply bin/getepel:7]: Sold.

File: paella/platform.py (hunk 2 above)
Comment (chayim) [reply paella/platform.py:331]: Great call. Kicking myself for not seeing this. Fixed in incoming.

**Review by rafie — APPROVED** [review]

**Review by rafie — APPROVED** [review]

**Pull request merged**